config/api-routes.js

```diff
@@ -1,6 +1,6 @@
 const API_PORT = 8085;
 
 const apiRoutes = {
-    '/api/integrations/': { host: `http://localhost:${API_PORT}` },
-    '/api/notifications/': { host: `http://localhost:${API_PORT}` }
+    '/api/user-preferences/': { host: `http://localhost:${API_PORT}` },
+    '/api/user-preferences/': { host: `http://localhost:${API_PORT}` }
```

Comment:
user preferences doesn't have a specific app to talk with. It can be plugged to any app (via email preferences) or to notifications (via notifications preferences)

Reply:
`'/api/user-preferences/notifications': {
    host: `http://localhost:${API_PORT}`,
  }`